src/faucet/interaction.ts

```diff
@@ -0,0 +1,38 @@
+import Entropy from "@entropyxyz/sdk"
+import { EntropyLogger } from '../common/logger'
+import { TESTNET_PROGRAM_HASH } from "./utils"
+import { EntropyFaucet } from "./main"
+import { print } from "src/common/utils"
+
+let chosenVerifyingKeys = []
+export async function entropyFaucet (entropy: Entropy, options, logger: EntropyLogger) {
+  const FLOW_CONTEXT = 'ENTROPY_FAUCET'
+  let faucetAddress
+  let chosenVerifyingKey
+  let verifyingKeys: string[] = []
+  const amount = "10000000000"
+  const { endpoint } = options
+  const selectedAccountAddress = entropy.keyring.accounts.registration.address
+  try {
+    const FaucetService = new EntropyFaucet(entropy, endpoint)
```

Comment:
Make this lowercase... you tricked me i thought these were all static methods
```suggestion
    const faucetService = new EntropyFaucet(entropy, endpoint)
```